cluster/gce/util.sh

```diff
@@ -1909,18 +1921,28 @@ function make-gcloud-network-argument() {
     fi
     ret="${ret},subnet=${subnetURL}"
     ret="${ret},aliases=pods-default:${alias_size}"
+    # if ${internal_address} is specified, attach it as an /32 alias so it's possible to
+    # detach it later.
+    if [[ -n ${internal_address:-} ]]; then
+      ret="${ret};${internal_address}"
+    fi
     ret="${ret} --no-can-ip-forward"
   else
+    ret="--network-interface"
     if [[ -n ${subnet:-} ]]; then
-      ret="${ret} --subnet ${subnetURL}"
+      ret="${ret} subnet=${subnetURL}"
     else
-      ret="${ret} --network ${networkURL}"
+      ret="${ret} network=${networkURL}"
     fi
-
-    ret="${ret} --can-ip-forward"
-    if [[ -n ${address:-} ]] && [[ "$address" != "no-address" ]]; then
-      ret="${ret} --address ${address}"
+    if [[ "${address:-}" == "no-address" ]]; then
```

Comment:
You said that you tested it only in ip-alias mode.
So do we really want to change that? Maybe it's enough to say that we don't support if IP-aliases are not supported - I would vote for that.

Reply:
I'm quite sure it didn't work in kubemark without this, but can't remember now exactly why. I will try to figure it out and remove if possible.